zetaclient/orchestrator/orchestrator.go

```diff
@@ -84,6 +89,7 @@ type multiLogger struct {
 func New(
 	ctx context.Context,
 	client interfaces.ZetacoreClient,
+	cometbftClient cometbft_rpc_client.Client,
```

Comment:
Let's integrate this into zetacore client wrapper in `zetaclient/zetacore/*`

Reply:
Isn't it better to ultimately have smaller focused interfaces rather than one giant interface.

Reply:
Generally yes, but here it introduces some unnecessary details to the orchestrator. Also mocking an interface for `oc.zetacore.NewBlockSubscriber(ctx) (chan ..., error)` is easier than for websocket client

Reply:
Ok I've moved it into the zetacore client.